serving/samples/gitwebhook-go/README.md

```diff
@@ -75,18 +75,18 @@ spec:
         spec:
           container:
             # Replace {DOCKER_HUB_USERNAME} with your actual docker hub username
-            image: docker.io/{DOCKER_HUB_USERNAME}/gitwebhook-go
+            image: docker.io/{DOCKER_HUB_USERNAME}/gitwebhook-go:latest
```

Comment:
Default tag is `:latest`, so I think we do not need put `:latest` to the end.

Reply:
seems fine, but aren't code changes needed for this?

Reply:
I think latest it redundant, but fine to have for clarity.

Code is fine.